docs/design/generics/terminology.md

```diff
@@ -74,11 +74,11 @@ parameter by parameter basis. A single function can take a mix of regular,
 generic, and template parameters.
 
 -   **Regular parameters**, or "dynamic parameters", are designated using the
-    "&lt;name>`:` &lt;type>" syntax (or "&lt;value>").
+    "\<name\>`:` \<type\>" syntax (or "\<value\>").
```

Comment:
What do you think of only escaping the `<`, not the `>`?

Reply:
Good?